app/.eslintrc.js

```diff
@@ -14,7 +14,12 @@ module.exports = {
     es2021: true,
   },
   parser: 'babel-eslint',
-  extends: ['plugin:react/recommended', 'standard', 'eslint-config-fbcnms'],
+  extends: [
+    'react-app',
```

Comment:
Curious about why this was added.  Can you leave a quick comment in this PR for posterity?

Reply:
Good point, forgot to mention it in the summary. Since we're using create-react-app, it's "highly recommend extending the base config, as removing it could introduce hard-to-find issues."

https://create-react-app.dev/docs/setting-up-your-editor/#extending-or-replacing-the-default-eslint-config